frontend/public/style/_vars.scss

```diff
@@ -27,8 +27,8 @@ $font-size-h3: var(--pf-global--FontSize--lg);
 $font-size-h4: var(--pf-global--FontSize--md);
 $font-size-h5: var(--pf-global--FontSize--md);
 $font-size-h6: var(--pf-global--FontSize--md);
-// Patternfly 3 defaults to 40px, which leaves a lot more whitespace than we're used to.
-$grid-gutter-width: 30px;
+// Patternfly 4 defaults to 24px
+$grid-gutter-width: 24px; // match var(--pf-global--spacer--lg)
```

Comment:
This change gives me pause.  We use `$grid-gutter-width` in a lot of places.

Reply:
$grid-gutter-width is mostly used for padding and margin.  I believe this change will fix the alignment.

Reply:
@gijohn, I recommend against this change.  Changing `$grid-gutter-width` will have widespread implications across the console, and we shouldn't be making changes of this nature after feature freeze.  cc:  @spadgett